### PR TITLE
Use only last_seen value to check if user is online

### LINKED ISSUE #428: last_seen is not set immediately after a newly registered user logs in
When a user registers and immediately logs in afterwards, their `last_seen` value will be null for the first 5 minutes of their account's existence. The logic for that is here: https://github.com/rainlab/user-plugin/blob/9113f843bdda6f98f1f5124af0809c52861edd67/models/User.php#L434

I know 5 minutes is a short time, but it actually already caused some confusion with users for a project I'm developing on. 

Possible solutions:
- Return created_at - 6 minutes as fallback in the [getLastSeen](https://github.com/rainlab/user-plugin/blob/master/models/User.php#L443) function so it'll trigger the `touchLastSeen` function immediately. 
- Set `last_seen` on `afterLogin`, but might cause impersonating users to set `last_seen` seen timestamps as well. 

Let me know your thoughts. I can help with a pull request if needed.

### PR DESCRIPTION
**Pull request by Samuell1**
Fixes #428

### CODE REVIEW

**Comment by LukeTowers** [comment]
This is changing the public API so it won't be accepted in its current state

**Comment by Samuell1** [comment]
@LukeTowers Yes i know but getLastSeen method isnt used anywhere just for this one thing.

**Comment by LukeTowers** [comment]
It's part of the public API, so there's no guarantee that it hasn't been used anywhere ever.

**Comment by Samuell1** [comment]
@LukeTowers Added back the method to be backwards compatible. Other changes should be okay.

**Comment by LukeTowers** [comment]
@Samuell1 is there any reason why you didn't go with the alternative solution of touching last_seen during the afterLogin event (after checking to make sure the user wasn't being impersonated)?

**Comment by Samuell1** [comment]
@LukeTowers Because this is simpliest version of fix adding more complex logic maybe will be better to add in other PR.

**Comment by bennothommo** [comment]
@Samuell1 I don't think touching the `last_seen` timestamp on `afterLogin` is too complex, though ;)

**Comment by Samuell1** [comment]
@bennothommo From commits it was there then suddenly it was removed but i dont see why.

**Comment by Samuell1** [comment]
@LukeTowers @bennothommo I think this should be good to go. Not sure why few years back the last_seen on login was removed its not even in commits history.

File: models/User.php [hunk 1]
```diff
@@ -460,7 +460,7 @@ public function touchLastSeen()
      */
     public function isOnline()
     {
-        return $this->getLastSeen() > $this->freshTimestamp()->subMinutes(5);
+        return $this->last_seen > $this->freshTimestamp()->subMinutes(5);
```
Comment (bennothommo) [inline models/User.php:463]: @Samuell1 I don't mind using `getLastSeen` here - we just simply needed the `last_seen` timestamp to be touched after login.

**Comment by daftspunk** [comment]
Fixed v1.5.5

**Pull request closed without merging**